integration/config/config_test.go

```diff
@@ -76,39 +76,46 @@ func TestConfigList(t *testing.T) {
 	assert.Check(t, is.DeepEqual(configNamesFromList(entries), testNames))
 
 	testCases := []struct {
+		desc     string
 		filters  filters.Args
 		expected []string
 	}{
-		// test filter by name `config ls --filter name=xxx`
 		{
+			desc:     "test filter by name",
 			filters:  filters.NewArgs(filters.Arg("name", testName0)),
```

Comment:
❤️ 

Guess we could already get these in separately, but not critical, so I'm ok with keeping it as part of this PR